Kickstarter-iOS/Assets.xcassets/icons/icon--heart.imageset/Contents.json

```diff
@@ -2,14 +2,11 @@
   "images" : [
     {
       "idiom" : "universal",
-      "filename" : "heart.pdf"
+      "filename" : "icon--heart.pdf"
     }
   ],
   "info" : {
     "version" : 1,
     "author" : "xcode"
-  },
-  "properties" : {
-    "template-rendering-intent" : "template"
```

Comment:
Not needed? I think adding this allows us to control tint color in code - I'm guessing in that case that the PDF was exported from Sketch with an existing color?

Reply:
All PDFs from Sketch should have an existing color (`soft-black` / `#282828` for most of them)